nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -246,6 +251,28 @@ void BtActionServer<ActionT>::executeCallback()
   }
 }
 
+template<class ActionT>
+void BtActionServer<ActionT>::populateErrorCode(typename std::shared_ptr<typename
+    ActionT::Result> result)
+{
+    // Grab error codes from blackboard
+    std::vector<int> error_codes;
+
+    // 0-99 error codes for follow path
+    error_codes.push_back(blackboard_->get<int>("follow_path_error_code"));
```

Comment:
You need to check if that's even possible to get it like you do below. This may not be set (in which case this throws an exception)

Reply:
Also use the parameterized name

Reply:
But see commentary below - this changes a bit